src/effects/presets/effectchainpreset.h

```diff
@@ -50,4 +56,5 @@ class EffectChainPreset {
     EffectChainMixMode::Type m_mixMode;
     double m_dSuper;
     QList<EffectPresetPointer> m_effectPresets;
+    bool m_readOnly = false;
```

Comment:
We normally don't use initialization in a header.  Please move that to the constructor.

Reply:
okido